README.md

```diff
@@ -1,2 +1,46 @@
-# TNW2
-TNW is a website with a number of articles showcasing trending stories, articles and innovation
+# TNW Website
+
+Second Project
+
+This is a project to demonstrate the existing knowledge of HTML and CSS. It is a individual project that shows the TNW Website
+
+The webpage is focused of showing positioning of elements using grid, flexbox and positioning.
+
+<img src="./images/Screenshot.png" alt="Screenshot">
+
+
+
+<i>Built With HTML5, CSS3, GIT, and VSCODE</i>
+
+Live Demo : [https://leon-mbegera.github.io/TNW2/.]
```

Comment:
## Improve the Live link display
- [ ] In my suggestion, we can make this link look neater. What do you think?
```suggestion
[Live Demo](https://leon-mbegera.github.io/TNW2/)
```